code/modules/client/preference_setup/loadout/loadout_eyes.dm

```diff
@@ -61,10 +59,32 @@
 	display_name = "Medical HUD, prescription (Medical)"
 	path = /obj/item/clothing/glasses/hud/health/prescription
 
-/datum/gear/eyes/shades
-	display_name = "Sunglasses, fat (Security/Command)"
-	path = /obj/item/clothing/glasses/sunglasses/big
-	allowed_roles = list("Security Officer","Head of Security","Warden","Colony Director","Head of Personnel","Quartermaster","Internal Affairs Agent","Detective")
+/datum/gear/eyes/medical/aviator
+	display_name = "Medical HUD Aviators (Medical)"
```

Comment:
Include that they added medical and engineering aviators to loadout for their HUD/mesons in the changes